app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+// const { getTreasures, getTreasuresSorted} = require('./controllers/treasures-controllers');
```

Comment:
Don't leave in commented out code...ESPECIALLY if it is from a different sprint.